pkg/cmd/server/api/v1/types.go

```diff
@@ -63,6 +63,9 @@ type NodeConfig struct {
 
 	// IPTablesSyncPeriod is how often iptable rules are refreshed
 	IPTablesSyncPeriod string `json:"iptablesSyncPeriod"`
+
+	// StreamingConnectionIdleTimeout is time out for streaming connections such as port forwarding
+	StreamingConnectionIdleTimeout string `json:"streamingConnectionIdleTimeout"`
```

Comment:
I would expect this under the serving stanza, like `maxRequestsInFlight` and `requestTimeoutSeconds` in the master config